dds/DCPS/DataReaderImpl.cpp

```diff
@@ -3282,6 +3282,56 @@ DataReaderImpl::get_ice_endpoint()
   return TransportClient::get_ice_endpoint();
 }
 
+DDS::ReturnCode_t DataReaderImpl::setup_deserialization()
```

Comment:
This still needs to be called. See where `DataWriterImpl` is calling `setup_serialization`.